project-base/assets/js/frontend/utils/showFormErrorsWindow.js

```diff
@@ -14,11 +14,11 @@ export default function showFormErrorsWindow (container) {
     if ($window.length === 0) {
         // eslint-disable-next-line no-new
         new Window({
-            content: $errorListHtml
+            errors: $errorListHtml
         });
-    } else {
-        $window.filterAllNodes('.js-window-validation-errors')
-            .html($errorListHtml)
-            .removeClass('display-none');
     }
+
+    $window.filterAllNodes('.js-window-validation-errors')
+        .html($errorListHtml)
+        .removeClass('display-none');
```

Comment:
This I don't understand, can you please explain to me
1. why is it necessary to move it outside the else branch?
2. why do we need this at all?

Reply:
1. Moved back into else block.
2. When we remove it the erros will dissapear on next enter key down.